rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -3191,6 +3194,204 @@ public void saveReceiveHeadersLastTimestamp_not_set() throws IOException {
         );
     }
 
+    @Test
+    public void getUtxoExpirationCheckpointRskBlockHeight_before_RSKIP264() {
+        Repository repository = mock(Repository.class);
+
+        when(repository.getStorageBytes(PrecompiledContracts.BRIDGE_ADDR, UTXO_EXPIRATION_CHECK_POINT_RSK_BLOCK_HEIGHT)).thenReturn(new byte[] { 1 });
+
+        BridgeStorageProvider provider = new BridgeStorageProvider(
+            repository, PrecompiledContracts.BRIDGE_ADDR,
+            config.getNetworkConstants().getBridgeConstants(), activationsBeforeFork
+        );
+
+        assertEquals(Optional.empty(), provider.getUtxoExpirationCheckpointRskBlockHeight());
+
+        verify(repository, never()).getStorageBytes(PrecompiledContracts.BRIDGE_ADDR, UTXO_EXPIRATION_CHECK_POINT_RSK_BLOCK_HEIGHT);
+    }
+
+    @Test
+    public void getUtxoExpirationCheckpointRskBlockHeight_after_RSKIP264() {
+        Repository repository = mock(Repository.class);
+
+        when(repository.getStorageBytes(PrecompiledContracts.BRIDGE_ADDR, UTXO_EXPIRATION_CHECK_POINT_RSK_BLOCK_HEIGHT)).thenReturn(new byte[] { 1 });
+
+        BridgeStorageProvider provider = new BridgeStorageProvider(
+            repository, PrecompiledContracts.BRIDGE_ADDR,
+            config.getNetworkConstants().getBridgeConstants(), activationsAllForks
+        );
+
+        assertEquals(Optional.of(1L), provider.getUtxoExpirationCheckpointRskBlockHeight());
+
+        verify(repository, atLeastOnce()).getStorageBytes(PrecompiledContracts.BRIDGE_ADDR, UTXO_EXPIRATION_CHECK_POINT_RSK_BLOCK_HEIGHT);
+    }
+
+    @Test
+    public void setUtxoExpirationCheckpointRskBlockHeight_after_RSKIP264() {
```

Comment:
I think we are missing a test for before RSKIP activation

Reply:
Resolved